packages/now-build-utils/src/prerender.ts

```diff
@@ -0,0 +1,38 @@
+import FileBlob from './file-blob';
+import FileFsRef from './file-fs-ref';
+import FileRef from './file-ref';
+import { Lambda } from './lambda';
+
+interface PrerenderOptions {
+  expiration: number;
+  lambda: Lambda;
+  fallback: FileBlob | FileFsRef | FileRef;
+}
+
+export class Prerender {
```

Comment:
I was calling these `Prerederables`, not the best name either but at least it's a noun

Reply:
👍  to `Prerederable`

👎 to `PrerenderFile` since it doesn't implement the `File` interface

Reply:
Personally (and @timneutkens), we see `Prerender` as a noun. So better, IMO.